onnxscript/values.py

```diff
@@ -136,7 +153,9 @@ def adapt_kwargs(self, kwargs):
     def __call__(self, *args, **kwargs):
         from onnxscript import evaluator  # pylint: disable=import-outside-toplevel
 
-        return evaluator.eval(self.opschema, args, kwargs)
+        # Hack:
+        # TODO(titaiwang): how to keep both graph_eval and eval if they share the same API?
```

Comment:
Related to https://github.com/microsoft/onnx-script/issues/297. But for now I guess you can context manage in exporter to change default evaluator via onnx-script api.